notebooks/hello-npu/hello-npu.ipynb

```diff
@@ -0,0 +1,1253 @@
+{
```

Comment:
<p>I think description in the brackets is not applicable for notebook, as it includes installation via pypi inside. It will be hard to maintain and track these changes apilcability from release to release and we can forgot to update it when other distribution channels will be also available</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1898/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>